modules/manually-create-identity-access-management.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/manually-creating-iam.adoc
+// * installing/installing_azure/manually-creating-iam-azure.adoc
+// * installing/installing_gcp/manually-creating-iam-gcp.adoc
+
+ifeval::["{context}" == "manually-creating-iam-aws"]
+:aws:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-gcp"]
+:google-cloud-platform:
+endif::[]
+
+[id="manually-create-iam_{context}"]
+== Manually create IAM
+
+The Cloud Credential Operator can be put into manual mode prior to installation
+in environments where the cloud identity and access management (IAM) APIs are
+not reachable, or the administrator prefers not to store an admin level
+credential secret in the cluster `kube-system` namespace.
+
+.Procedure
+
+. Run the {product-title} installer to generate manifests:
++
+----
+$ openshift-install create manifests --dir=mycluster
+----
+
+. Insert a ConfigMap into the manifests directory so that the Cloud Credential
+Operator will be be placed in manual mode:
++
+----
+$ cat <<EOF > mycluster/manifests/cco-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-credential-operator-config
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    release.openshift.io/create-only: "true"
+data:
+  disabled: "true"
+EOF
+----
+
+. Remove the admin credential secret created using your local cloud credentials.
+This removal prevents your admin credential from ever being stored in the
+cluster:
++
+----
+$ rm mycluster/openshift/99_cloud-creds-secret.yaml
+----
+
+. Obtain the {product-title} release image your `openshift-install` binary is built
+to use:
++
+----
+$ bin/openshift-install version
+release image quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64
+----
+
+. Locate all `CredentialsRequests` in this release image that target the cloud you
+are deploying on:
++
+----
+$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64 --to ./release-image
+----
+
+. Locate the `CredentialsRequests` in the extracted file:
++
+----
+$ grep -l "apiVersion: cloudcredential.openshift.io" * | xargs cat
```

Comment:
There will be lots of output here, I'm afraid that user may make mistake when finding the CR text
so IMO, beside above command, it is better that list all cloud provider's CR here for user reference.
@dgoodwin @joelddiaz WDYT?

Reply:
I would not want to embed all the credentials details in the documentation as that would be very awkward to maintain release by release. 

We have however implemented an oc adm command to simplify this in 4.6 so in very near future we could update these docs to avoid any mistakes in this portion. At present I think this may be the best option for now, users are opting into the possibility of making mistakes in moving to manual credentials management. We hope to make it easier with additional tooling in future.

Reply:
Right, oc tool is a great option for users.